src/app/messages-page/messages-inbox/messages-filter-menu/messages-filter-menu.component.html

```diff
@@ -52,7 +52,7 @@
           id="button-basic"
           dropdownToggle
           type="button"
-          class="btn btn-secondary fs-16px dropdown-toggle text-grey6"
+          class="btn btn-dark fs-16px dropdown-toggle text-grey6"
```

Comment:
the `btn-dark` class makes this dropdown look disabled in dark and icy dark mode: 
![image](https://user-images.githubusercontent.com/81658138/121787667-44778980-cb7c-11eb-8de9-99d72afa718e.png)

Reply:
Yea true debated just leaving it, so will revert to secondary color